doc/src/getting-started-guide.md

```diff
@@ -209,11 +209,154 @@ Once the application has been flashed, you may connect to the ESP32 over the ser
 
 ## Getting Started on the STM32 platform
 
-> TODO will se distribute a binary image for STM32?
+AtomVM can run on a wide variety of STM32 chipsets available from [STMicroelectronics](https://www.st.com). The support is not nearly as mature as for the ESP32 platform, but work is ongoing, and pull requests are always welcome. At this time AtomVM will work on any board with a minimum of around 128k ram and 512k (1M recommended) flash. Simple applications and tests have been successfully run on a stm32f411ceu6 (A.K.A. Black Pill V2). These minimum requirements may need to be raised as platform support matures.
+
+### Prerequisites
+
+* [st-flash](https://github.com/texane/stlink), to flash both AtomVM and your packed AVM applications. Make sure to follow its [installation procedure](https://github.com/texane/stlink#installation) before proceeding further.
+* arm-none-eabi/arm-elf toolchain compatible with your system
+* `cmake`
+* `make`
+* A serial console program, such as `minicom` or `screen`, so that you can view console output from your AtomVM application.
+
+### Building
+Before building for the first time you need to have a compiled clone of the libopencm3 libraries, from inside the AtomVM/src/platforms/stm32 directory:
```

Comment:
I did think about splitting it up, but since getting started on the stm32 is a bumpier road, I wanted to keep it easier on the user and not make them jump all over the place to get to blinky. These are all of the steps necessary to "get started" I wanted to keep them together. Due to the fact that our "target" board is obsolete and unavailable for purchase, and the cpu clock and peripheral clock configurations, UART pin assignment, and other settings need to be changed to match each individual board configuration (there are 100's, counting third party designs using stm32 chips that AtomVM can run on) it seems unrealistic to build images in the foreseeable future. We may want to pick a new `target` device that is available and reasonably affordable.

Maybe I just need to add more to the top level `README.STM32.Md` so it is all in one place and go ahead and split the sections here.

Reply:
ok